master/security/service-accounts.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Use service accounts in policy rules
+---
+
+### Big picture
+
+Use Calico network policy to allow/deny traffic for Kubernetes services. 
+
+### Value
+
+Using Calico network policy, you can leverage Kubernetes service accounts with RBAC for flexible control over how policies are applied in a cluster. For example, the Network Security can have RBAC permissions to:
+
+- Control which service accounts the Developer team can use within a namespace
+- Write high-priority network policies for those service accounts (that the Developer team cannot override) 
+
+The Network Security team can maintain full control of security, while selectively allowing Developer operations where it makes sense.  
+
+Using **Istio-enabled apps** with Calico network policy, the cryptographic identity associated with the service account is checked (along with the network identity) to achieve two-factor authentication.
+
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+**NetworkPolicy** or **GlobalNetworkPolicy** with a service account rule and match criteria.
+
+
+### Concepts
+
+#### Use smallest set of permissions required
+
+Service accounts are controlled by RBAC so you can grant permissions to trusted entities (code and/or people). To perform any operation in a workload, clients are required to authenticate with the Kubernetes API server. If you do not explicitly assigned a service account to a pod, it uses the default ServiceAccount in the namespace. You should not grant broad permissions to the default service account for a namespace. If an application needs access to the Kubernetes API, create separate service accounts with the smallest set of permissions required. 
+
+### Before you begin...
+
+Configure Kubernetes service accounts with RBAC
```

Comment:
Worth a link to the Kubernetes documentation for this?

> Configure Kubernetes service accounts with RBAC

Also, I don't think the RBAC part is relevant for this doc - really the prereq is just "Configure unique Kubernetes service accounts for your applications" or something like that, right?

Reply:
Fixed.